### ISSUE TITLE
Volt Macro Partial Output 2.1.0R

### ISSUE THREAD

**Issue by Smoozilla** [phalcon/cphalcon#11857]
```
{%- macro panel(title, body) %}
    <div class="col-md-3">
        <div class="panel panel-default center">
            <div class="panel-heading"><strong>{{ title }}</strong></div>
            <div class="panel-body">
                {{ body }}
            </div>
        </div>
    </div>
{%- endmacro %}
```

```
<h2 class="sub-header">Edit Account</h2>
    <div class="table-responsive">
        <div class="panel panel-primary">
            <div class="panel-heading">{{ account.email }}</div>
            <div class="panel-body">
                {{ panel("ID", account.id) }}
                {{ panel("E-Mail", account.email) }}
                {{ panel("Last Login", account.lastlogin) }}
                {{ panel("Created IP", account.created_ip) }}
                {{ panel("Role", account.role) }}
            </div>
        </div>
</div>
```

Produces

```
<h2 class="sub-header">Edit Account</h2>
    <div class="table-responsive">
       <div class="panel panel-primary">
          <div class="panel-heading">me@gmail.com</div>
          <div class="panel-body">
             <div class="col-md-3">
                <div class="panel panel-default center">
                   <div class="panel-heading"><strong>I</strong></div>
                   <div class="panel-body">
                      D        
                   </div>
                </div>
             </div>
             <div class="col-md-3">
                <div class="panel panel-default center">
                   <div class="panel-heading"><strong>E</strong></div>
                   <div class="panel-body">
                      -        
                   </div>
                </div>
             </div>
             <div class="col-md-3">
                <div class="panel panel-default center">
                   <div class="panel-heading"><strong>L</strong></div>
                   <div class="panel-body">
                      a        
                   </div>
                </div>
             </div>
             <div class="col-md-3">
                <div class="panel panel-default center">
                   <div class="panel-heading"><strong>C</strong></div>
                   <div class="panel-body">
                      r        
                   </div>
                </div>
             </div>
             <div class="col-md-3">
                <div class="panel panel-default center">
                   <div class="panel-heading"><strong>R</strong></div>
                   <div class="panel-body">
                      o        
                   </div>
                </div>
             </div>
          </div>
       </div>
    </div>
</div>
```

**Comment by Smoozilla** [comment]
Also see

https://forum.phalconphp.com/discussion/11727/volt-macro-bug

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch.

**Comment by sergeyklay** [comment]
Fixed in the `2.1.x` branch.
